css/pages/index.css

```diff
@@ -19,22 +19,28 @@
 
 .card {
   border: 1px solid #f3efef;
-  box-shadow: 0 1.5px 10px 0 rgba(0, 0, 0, 0.14);
+  box-shadow: 0 0 15px -9px rgba(0, 0, 0, 0.65);
   border-radius: 2px;
   width: 30%;
   padding: 18px 15px;
   display: flex;
   justify-content: center;
   align-items: center;
-}
-
-.card a {
+  cursor: pointer;
```

Comment:
Do we really this need this? Anchor tags provide this by default, I suppose. Can you please cross check?

Reply:
> Do we really this need this? Anchor tags provide this by default, I suppose. Can you please cross check?

Oh yeah I forgot to remove it ! I added this when we were using div tag